### PR TITLE
Implement script for updating devextreme-cli template files

### PR DESCRIPTION
**Pull request by egaluza**

### CODE REVIEW

File: templates-generator/index.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+const platformsConfigs = {
+    react: './react-config.js'
+};
+
+const commands = args['_'];
+(() => {
+    if(commands.length) {
+        console.error('Command is redundant');
+        return;
```
Comment (tar-z) [inline templates-generator/index.js]: I suggest to exit with not success  code in this case

File: templates-generator/index.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    }
+
+    function createNestedFolder(fullPath) {
+        const dirName = path.dirname(fullPath);
+        if(!fs.existsSync(dirName)) {
+            fs.mkdirSync(dirName, { recursive: true });
+        }
+    }
+
+    process.exit();
```
Comment (tar-z) [inline templates-generator/index.js]: Do we need to call it here?

Comment (egaluza) [reply templates-generator/index.js]: Replace it

File: templates-generator/index.js (hunk 1 above)
Comment (egaluza) [reply templates-generator/index.js]: Fix

**Pull request merged**